src/components/SlateEditor/plugins/comment/inline/index.tsx

```diff
@@ -37,27 +37,8 @@ export const commentInlineSerializer: SlateSerializer = {
   },
 };
 
-const onBackspace = (e: KeyboardEvent, editor: Editor, nextOnKeyDown?: (event: KeyboardEvent) => void) => {
-  // Replace comment with paragraph if last character is removed
```

Comment:
Ser ikkje heilt behovet for dette for inlineembeds som forklaring og kommentar. Kjør debatt.